### ISSUE TITLE
Select opens only after blur and refocus

### ISSUE THREAD

**Issue by d-lamers** [selectize/selectize.js#624]
When I type something nothing shows up, then I blur and focus and then the dropdown shows and I can select an item. (This is with an ajax call)

**Comment by d-lamers** [comment]
Wondering if someone is working on this issue?

**Comment by HadrienPierart** [comment]
I'm experiencing the same issue, did someone find a solution or a cause to this ?
It's also with data fetched through an ajax call. I'm thinking that there might be a link with a digest cycle not properly completed on the dropdown after getting the data through ajax. Investigating...

**Comment by HadrienPierart** [comment]
For future reference and because it took me 2h to figure out the issue on my part : 
Everything was working ok on Chrome. Only Firefox seemed to fail to filter the list when I clicked and typed some text. However if I clicked a second time in the input, I could then make it work normally. Which lead me investigate a issue related to focus / blur on the component.
As a matter of fact, I have another directive interfering with the focus of the underlying input of the select. As soon as I removed it / modified the behavour of my focus listener, the select worked as expected in Firefox !
All good for me. Hope this helps.
